src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/ProjectConfigurationsInferredFromUsageCapabilitiesProvider.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.ComponentModel.Composition;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.VisualStudio.ProjectSystem
+{
+    [Export(ExportContractNames.Scopes.UnconfiguredProject, typeof(IProjectCapabilitiesProvider))]
+    [AppliesTo(ProjectCapabilities.AlwaysApplicable)]
+    internal sealed class ProjectConfigurationsInferredFromUsageCapabilitiesProvider : UnconfiguredProjectCapabilitiesProviderBase
```

Comment:
It needs to be AlwaysApplicable because there's no other capability in play that we can hook onto - this is called before  ActiveConfig has been determined. CPS are going to work on the real fix tomorrow.